keps/sig-network/1435-mixed-protocol-lb/README.md

```diff
@@ -476,6 +476,20 @@ Kube-proxy will not block traffic based on the port status information from `Ser
 
 ### Test Plan
 
+[x] I/we understand the owners of the involved components may require updates to
+existing tests to make this code solid enough prior to committing the changes necessary
+to implement this enhancement.
+
+##### Prerequisite testing updates
+
+##### Unit tests
+
+Proper unit tests are in place for the affected code parts.
+
+##### Integration tests
+
+##### e2e tests
+
 There must be e2e cases that test whether CPI implementations handle Service definitions with mixed protocol configuration on a consistent way. I.e. either the cloud LB is set up properly or the Service is rejected by the CPI implementation.
```

Comment:
Are we still expecting these e2e tests as a requirement to GA? Given this is a provider-specific test, an integration test validating the new API behavior might be sufficient. Providers should test this though if they support mixed protocols.

Reply:
I would be in favor of requiring integration tests for GA and removing e2e test requirement.

Reply:
I have updated the e2e test part according to your suggestion.

Reply:
I think we should have e2e for GA, the API change and provider specific arguments are right , but there is also a behaviour that needs to be verified on the Service proxies implementations that need to be covered, we can't assume that it will work for all implementations since some of them treat LoadBalancer services traffic differently https://github.com/kubernetes/enhancements/pull/3587#discussion_r988810889

Reply:
@andrewsykim @aojea I updated the e2e part.

Reply:
Thanks, it should not be hard, there are already some e2e LoadBalancer tests that can be used as template

ping me if you need help